services/ui-src/src/forms/internal/abf.schema.ts

```diff
@@ -12,5 +10,15 @@ export default object({
     .matches(dateFormatRegex, "Invalid start date"),
   "abf-endDate": string()
     .required("End date is required")
-    .matches(dateFormatRegex, "Invalid end date"),
+    .matches(dateFormatRegex, "Invalid start date")
+    .test(
+      "is after start date",
+      "End date cannot be before start date",
+      (endDateString, context) => {
+        const startDateString = context.parent["abf-startDate"];
+        const startDate = new Date(startDateString);
+        const endDate = new Date(endDateString!);
+        return endDate >= startDate;
+      }
+    ),
```

Comment:
This is the new test for checking startDate vs. endDate. Had to do it this way to play nicely with our new string type.